.github/workflows/pipeline-deploy-app-preprod-prod.yaml

```diff
@@ -0,0 +1,77 @@
+name: Pipeline Deployment Application production & pre-production Environment
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Select the Tag to deploy artefact from"
+        required: true
+        type: string
+      environment:
+        description: "Select the deployment environment"
+        required: true
+        type: string
+        options:
+          - preprod
+          - prod
+jobs:
+  metadata:
+    name: "Get Metadata"
+    uses: NHSDigital/uec-dos-management/.github/workflows/metadata.yaml@v0.42
+
+  derive-artefact-address:
+    name: "Get artefact location"
+    uses: NHSDigital/uec-dos-management/.github/workflows/derive-artefact-address.yaml@v0.42
+    with:
+      tag: ${{ inputs.tag }}
+
+  deploy-application-infrastructure:
+      needs:
+        [
+          metadata,
+        ]
+      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@v0.42
+      with:
+        environment: ${{ inputs.environment }}
+        domain: "uec-cm"
+        project: cm
+        workspace: default
+        stacks: '["application", "ui-test"]'
```

Comment:
Do we need to deploy ui test to prod or preprod

Reply:
Have removed ui-test stack